src/main/java/org/folio/orders/utils/ErrorCodes.java

```diff
@@ -41,7 +41,8 @@
   COST_ADDITIONAL_COST_INVALID("costAdditionalCostInvalid", "Cost's additional cost must be positive number"),
   COST_DISCOUNT_INVALID("costDiscountInvalid", "Cost's discount is invalid"),
   LOC_NOT_PROVIDED("locNotProvided", "Location not provided"),
-  ORGANIZATION_NOT_A_VENDOR("organizationNotAVendor", "Order cannot be opened as the associated vendorId belongs to an organization not vendor");
+  ORGANIZATION_NOT_A_VENDOR("organizationNotAVendor", "Order cannot be opened as the associated vendorId belongs to an organization not vendor"),
```

Comment:
don't feel like you have to change this but "belongs to a non-vendor organization" sounds better to me

Reply:
And yes I realize this isn't even something you're changing here... I just noticed it that's all

Reply:
ofcourse, changed it